### ISSUE TITLE
Error when using dismissable tip UI element

### ISSUE THREAD

**Issue by d-karstens** [verbb/vizy#235]
### Describe the bug

Hi,

We've encountered a bug where a block doesn't display properly if it contains a dismissable tip UI element

I've attached screenshots of our field setup and the js error shown my browser console

![Screenshot 2023-06-23 at 9 55 50 am](https://github.com/verbb/vizy/assets/21274925/423e9874-ac79-4f01-ad6f-59edbed6e4d8)
![Screenshot 2023-06-23 at 9 58 16 am](https://github.com/verbb/vizy/assets/21274925/3e035dab-f850-4a49-a1a0-234881f18271)

### Steps to reproduce

1. Create a block with a tip UI element that has the dismissable option ticked
2. Try to add the block in an entry

### Craft CMS version

4.4.14

### Plugin version

2.1.6

### Multi-site?

No

**Comment by engram-design** [comment]
Yes, that's a tricky one, because it's outputting `<script>` tags inline with content, which will throw an error in Vue. Fields handle this is a particular way, so I'll need to do some digging to see what we can do for UI elements.
